packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2.ts

```diff
@@ -17,6 +17,27 @@ export interface SecondaryAddressProps {
   readonly cidrBlockName: string;
 }
 
+/**
+ * Additional props needed for BYOIP IPv6 address props
+ */
+export interface Ipv6PoolSecondaryAddressProps extends SecondaryAddressProps {
+  /**
+   * ID of the IPv6 address pool from which to allocate the IPv6 CIDR block
+   * Note: BYOIP Pool ID is different than the pool ID of IPAM.
```

Comment:
```suggestion
   * Note: BYOIP Pool ID is different from the IPAM Pool ID.
```
Feel free to disregard this if the previous wording was intentional as I'm not sure exactly what IPAM pool ID prop it's referring to.